src/main/java/com/exadel/project/internship/controller/UnpostedInternshipController.java

```diff
@@ -0,0 +1,66 @@
+package com.exadel.project.internship.controller;
+
+
+import com.exadel.project.common.exception.EntityNotFoundException;
+import com.exadel.project.common.service.S3Service;
+import com.exadel.project.internship.dto.InternshipDTO;
+import com.exadel.project.internship.dto.InternshipDetailsDTO;
+import com.exadel.project.internship.service.InternshipService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.io.IOException;
+import java.util.List;
+
+@RestController
+@RequestMapping("unposted-internship")
```

Comment:
/unposted-internship ?

Reply:
fixed by single controller with separated url for interns and admins